src/components/tipRecords/TipMedia.vue

```diff
@@ -1,106 +1,91 @@
 <template>
-  <div class="tip-media">
-    <ImageGallery
-      v-if="openGallery"
-      :media="media"
-      @close="openGallery = false"
-    />
-    <div
-      class="media-container"
-      :class="`template${Math.min(media.length, 4)}`"
-      @click.stop="openGallery = true"
+  <div
+    class="tip-media"
+    :class="`template${Math.min(media.length, 4)}`"
+  >
+    <img
+      v-for="(image, index) in media.slice(0, 4)"
+      :key="index"
+      :style="{ 'grid-area': `pos${index}` }"
+      :src="image"
+      loading="lazy"
+      @click.stop="openImageGallery(index)"
+      @error="$event.target.src = defaultImage"
     >
-      <img
-        v-for="(image, index) in media.slice(0,4)"
-        :key="index"
-        :style="{ 'grid-area': `pos${index}` }"
-        :src="image"
-        loading="lazy"
-        @error="$event.target.src = require('../../assets/defaultImg.svg')"
-      >
-      <span
-        v-if="media.length > 4"
-        class="show-more"
-      >{{ `+${media.length - 4}` }}</span>
-    </div>
+    <span
+      v-if="media.length > 4"
+      class="show-more"
+    >{{ `+${media.length - 4}` }}</span>
   </div>
 </template>
 
 <script>
-import ImageGallery from '../ImageGallery.vue';
+import defaultImage from '../../assets/defaultImg.svg';
 
 export default {
-  components: { ImageGallery },
   props: {
     media: { type: Array, required: true },
   },
-  data: () => ({ openGallery: false }),
+  data: () => ({ defaultImage }),
+  methods: {
+    openImageGallery(imageIndex) {
+      this.$store.dispatch('modals/open', {
+        name: 'image-gallery', media: this.media, imageIndex,
+      });
+    },
+  },
 };
 </script>
 
 <style lang="scss" scoped>
 .tip-media {
-  padding-top: 55%;
+  max-height: 210px;
```

Comment:
@AtanasKrondev how height should be actually defined? I can't get it from the code

Reply:
According to the design it should be 16:9 ratio.

Reply:
```suggestion
  aspect-ratio: 16/9;

  @supports not (aspect-ratio: 16/9) {
    max-height: 360px;
    @include smallest {
      max-height: 210px;
    }
  }
```

<a href="https://caniuse.com/mdn-css_properties_aspect-ratio" target="_blank">Compatibility</a>. Honestly I didn't know about the ```aspect-ratio``` property and I don't vouch for it but seems suitable in this case.🤷🏻‍♂️

Reply:
also in the failback, we can define the proper height for each breakpoint, check the last commit